### PR TITLE
chore(mobile): discovery animation

### PR DESCRIPTION
**Pull request by Nodonisko**
## Related Issue

Resolve 

## Screenshots:

https://satoshilabs.slack.com/files/U02RLDSFCPP/F06ADFG81RR/1000008533.mp4

### CODE REVIEW

File: suite-native/assets/src/components/DiscoveryAssetsLoader.tsx [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
-const EMPTY_LIST_SKELETON_COUNT = 3;
-const NONEMPTY_LIST_SKELETON_COUNT = 1;
+const DEFAULT_LIST_SKELETON_COUNT = 3;
+const MINIMUM_LIST_SKELETON_COUNT = 1;
 
-export const DiscoveryAssetsLoader = () => {
+export const DiscoveryAssetsLoader = ({ numberOfAssets }: { numberOfAssets: number }) => {
     const { translate } = useTranslate();
-    const isListEmpty = useSelector(selectIsAccountsListEmpty);
+    const isListEmpty = numberOfAssets === 0;
```
Comment (juriczech) [inline suite-native/assets/src/components/DiscoveryAssetsLoader.tsx:12]: Please use `selectIsPortfolioEmpty` so it's consistent. Any reason why not?

Comment (Nodonisko) [reply suite-native/assets/src/components/DiscoveryAssetsLoader.tsx:12]: It returns false if discovery is in progress which is not desired here

Comment (juriczech) [reply suite-native/assets/src/components/DiscoveryAssetsLoader.tsx:12]: And why not `selectIsAccountsListEmpty`? Same thing without the discovery check

Comment (Nodonisko) [reply suite-native/assets/src/components/DiscoveryAssetsLoader.tsx:12]: I need `numberOfAssets` anyway for other calculations so I don't see point introduction another selector to this component if can do simple check like this

**Review by juriczech — APPROVED** [review]

**Pull request merged**